spellings/getFiles

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+bashPass="\033[32;1mPASSED -"
+bashInfo="\033[33;1mINFO -"
+bashFail="\033[31;1mFAILED -"
+bashEnd="\033[0m"
+
+# Check number of arguments
+files=""
+file=""
+excludeDirs=""
+excludeFiles=""
+includeExtensions=""
+
+# Make the assumption that Mac users will be using FreeBSD Get Opt
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    while [ $# -gt 0 ]; do
+    case "$1" in
+        -ed= | --exclude-dirs=* )
+            excludeDirs="-E $(echo "${1#*=}" | sed -E 's/,/ -E /g')"
+            shift
+            ;;
+        -ef= | --exclude-files=* )
+            excludeFiles="-E $(echo "${1#*=}" | sed -E 's/,/ -E /g')"
+            shift
+            ;;
+        -ie= | --include-extensions=* )
+            includeExtensions="-e $(echo "${1#*=}" | sed -E 's/,/ -e /g')"
+            shift
+            ;;
+        -h | --help )
+                echo -e "$bashInfo Find all .c and .h files with the Amazon copyright in them $bashEnd"
```

Comment:
Nit-pick: Perhaps another scripting language would be more suitable here.

Reply:
I honestly am debating it, I was sticking to this one for some reason and it's been a huge headache getting it to work for mac and linux and I think the mac implementation is actually buggy...
I was thinking of using golang?

Reply:
Ended up changing the way we get files for this, marking it resolved